src/PollController.js

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) 2020, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
```

Comment:
why don't we use `import` pattern for new files we add ?

Reply:
yeah, mostly for consistency. None of the v1 files use the import pattern. We'hv made sure the `v2` stuff uses the import pattern.

Reply:
We'll probably do this using a script and dedicate QA time to make sure we dont break any embedded browser flows

Reply:
Cool. We should prioritize this in near future, but i understand the impact and need of comprehensive testing.